### PR TITLE
Stream guides minor fixes

### PR DESCRIPTION
**Pull request by bono007**
This is a proposal to improve a few minor issues I ran into while recently going through the guides.

Typically I would split these into separate pull requests but am not sure of the desired granularity here in the microsite repo. Just let me know and I will gladly split them out if need be.

### CODE REVIEW

File: content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-sample.md [hunk 1]
```diff
@@ -75,11 +75,11 @@ In this step, we create the `UsageDetailSender` source.
 
 <!--Kafka-->
 
-You can either [directly download the project generated by the Spring Initialzr](https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.4.3.RELEASE&packaging=jar&jvmVersion=1.8&groupId=io.spring.dataflow.sample&artifactId=usage-detail-sender-kafka&name=usage-detail-sender-kafka&description=Demo%20project%20for%20Spring%20Boot&packageName=io.spring.dataflow.sample.usagedetailsender&dependencies=cloud-stream,actuator,kafka)
+You can either [directly download the project generated by the Spring Initialzr](https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.4.12&packaging=jar&jvmVersion=1.8&groupId=io.spring.dataflow.sample&artifactId=usage-detail-sender-kafka&name=usage-detail-sender&description=Demo%20project%20for%20Spring%20Boot&packageName=io.spring.dataflow.sample.usagedetailsender&dependencies=cloud-stream,actuator,kafka)
```
Comment (bono007) [inline content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-sample.md:78]: All of the Initializr links were updated w/ the following changes:

- 2.4.3.RELEASE -> 2.4.12 as the former causes a big warning popup about 2.4.12 being the latest 2.4x release.
- Remove the binder specific suffix from the generated app in order to match the class names in the example test code which does not have a `(Kafka|Rabbit)Application` suffix.

File: content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-sample.md [hunk 2]
```diff
@@ -75,11 +75,11 @@ In this step, we create the `UsageDetailSender` source.
...
 
 or visit the [Spring Initialzr site](https://start.spring.io/) and follow these instructions:
 
-1. Create a new Maven project with a Group name of `io.spring.dataflow.sample` and an Artifact name of `usage-detail-sender-kafka`, and package `o.spring.dataflow.sample.usagedetailsender`.
+1. Create a new Maven project with a Group name of `io.spring.dataflow.sample` and an Artifact name of `usage-detail-sender-kafka`, and package `io.spring.dataflow.sample.usagedetailsender`.
```
Comment (bono007) [inline content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-sample.md:82]: I made this change under the assumption that the package of `o.spring...` was a typo and should be `io.spring...`

File: content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-sample.md [hunk 3]
```diff
@@ -234,7 +234,7 @@ public class UsageDetailSenderApplicationTests {
 				.run()) {
 
 			OutputDestination target = context.getBean(OutputDestination.class);
-			Message<byte[]> sourceMessage = target.receive(10000);
+			Message<byte[]> sourceMessage = target.receive(10000, "usage-detail");
```
Comment (bono007) [inline content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-sample.md:237]: Updated as per https://github.com/spring-cloud/spring-cloud-stream/blob/afe8045493b04303ccd890a3f893abf10c881d0e/spring-cloud-stream/src/test/java/org/springframework/cloud/stream/binder/test/OutputDestination.java#L85

**Comment by cppwfs** [comment]
LGTM

**Pull request merged**